tendrl/commons/flows/import_cluster/__init__.py

```diff
@@ -15,6 +16,27 @@
 
 
 class ImportCluster(flows.BaseFlow):
+    def _probe_and_mark_provisioner(self, node_list, integration_id):
+        try:
+            gluster_prov_nodes = json.loads(
+                NS._int.client.read("indexes/tags/provisioner/gluster").value
+            )
+            if gluster_prov_nodes and len(gluster_prov_nodes) > 0:
+                # Ideally only one node would be present in this list
+                # Even if more than one in the list, mark one of them
+                # with tag `provisioner/{integration-id}` for further
+                # cluster operations
+                nc = NS.tendrl.objects.NodeContext(
+                    node_id=gluster_prov_nodes[0]
```

Comment:
I had requested to check if gluster_prov_nodes[0] is in node_list, this current code can also add a completely different cluster node , so make that check